test/gitea_access_control_test.go

```diff
@@ -167,8 +169,9 @@ func TestGiteaACLOrgAllowed(t *testing.T) {
 		YAMLFiles: map[string]string{
 			".tekton/pr.yaml": "testdata/pipelinerun.yaml",
 		},
-		NoCleanup:    true,
-		ExpectEvents: false,
+		NoCleanup:            true,
```

Comment:
Do we want to keep this true by default ? 
asking because it will not cleanup if its true

Reply:
it's a old field not used, but that's not the point of this pr, you can send a cleanup pr if you want later on.

Reply:
removed them as part of a commit